tests/sftp/sensors/test_sftp.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from airflow.exceptions import TaskDeferred
+
+from astronomer.providers.sftp.sensors.sftp import SFTPSensorAsync
+from astronomer.providers.sftp.triggers.sftp import SFTPTrigger
+
+
+def context():
+    """Creates an empty context."""
+    context = {}
+    yield context
+
+
+def test_sftp_run_now_sensor_async():
```

Comment:
can you please convert these tests to class-based tests. you can take a look at either core or cncf tests https://github.com/astronomer/astronomer-providers/blob/main/tests/core/sensors/test_external_task.py